src/languages/yaml.js

```diff
@@ -20,12 +20,12 @@ export default function(hljs) {
   const KEY = {
```

Comment:
Should we add a look-ahead for `:` at the end perhaps to make sure these are keys?

Reply:
@joshgoebel Should I add it if I think I need it? How should I do it